docs/src/authorizations_processors/execution_environment_differences.md

```diff
@@ -0,0 +1,15 @@
+# Execution Environment Differences
+
+Depending on the type of `Execution Environment` being used, the behavior of the `Processor` may vary. In this section we will describe the main differences in how the `Processor` behaves in the different `Execution Environments` that we support.
+
+### Execution Success
+
+During the execution of a `Message Batch`, the `Processor` will execute each function of the subroutine of that batch. If the execution for a specific function fails, we will consider the execution failed in case of `Atomic` batches, and we will stop the execution of the next function in case of `NonAtomic` batches.
+
+Currently, in the `CosmWasm` Execution Environment, a function fails if the `CosmWasm` contract that we are targeting doesn't exist, if the `entry point` of that contract doesn't exist, or if the execution of the contract fails for any reason. On the contract, in the `EVM` Execution Environment, a function only fails if the contract explicitly fails or reverts.
+
+To mitigate the differences in behavior between these two `Execution Environments`, we have added a check in the `EVM` Processor version to consider the execution failed if the contract doesn't exist by explicitly checking if the contract indeed exists. We've also added a revert in our `EVM` libraries if the execution of the contract enters the fallback function, which is not allowed in our system. Nevertheless, since the `Processors` are not restricted to `Valence Libraries` but can call any contract, we can't guarantee that the contract targeted will fail if an entry point doesn't exist, because the fallback function might not be defined or might not revert.
```

Comment:
```suggestion
To mitigate the differences in behavior between these two execution environments, a an `EVM` Processor version check was included to check if the contract indeed exists and fail execution if the contract does not exist. Behavior was also added in the `EVM` libraries to revery if the execution of the contract enters the fallback function, which is not allowed in the system. Nevertheless, since `Processors` are not restricted to `Valence Libraries` but can call any contract, no guarantee can be made that the contract targeted will fail if an entry point does not exist, because the fallback function might not be defined or might not revert.
```